server/jetstream_cluster.go

```diff
@@ -5308,6 +5318,23 @@ func (s *Server) jsClusteredConsumerRequest(ci *ClientInfo, acc *Account, subjec
 				break
 			}
 		}
+		if check {
```

Comment:
If we placed the stream but now can't place the consumer we can't overflow a consumer, they have to overlay the same peer set. If this is just moving to new peer that is ok, the one inherited from the parent stream.

Reply:
are you saying, don't check the limit for consumer?

Reply:
Not saying that, just saying we can't overflow a consumer to a new cluster.

Reply:
So we would have to fail the consumer create at that point..

Reply:
That's what this does.